static/src/App.vue

```diff
@@ -66,12 +76,18 @@ export default defineComponent({
         id="login-view"
         @login="login"
       ></LoginView>
+      <TasksOverView
+        v-else-if="page == 'tasks'"
+        id="tasks-overview"
+        @annotate="annotate"
+      ></TasksOverView>
       <TaskView
-        v-else
+        v-else-if="page == 'task' && task"
```

Comment:
They way we handle things at the moment means that everything is displayed at the same base URL, right? So the browser-back/forth-buttons are useless and (except for logout) we don't have any "control" buttons yet. 
So right now, its impossible to get back to the overview after you clicked on a task, except for reloading the entire page or logging out.

Definitely shouldn't change that in this PR but maybe something that we should think about, as we have 3 completely different things to display on the same page.